stl/inc/random

```diff
@@ -4163,63 +4301,70 @@ public:
     struct param_type { // parameter package
         using distribution_type = student_t_distribution;
 
-        param_type() {
+        param_type() noexcept // strengthened
+        {

```

Comment:
Most lines are using `noexcept /* strengthened */` to avoid detaching the brace (good), but some lines are using `// strengthened` which forces the brace to the next line. Can these be made consistent?

Reply:
I used to think the convention is using `// strengthened` for constructors. Now I'm changing to use `noexcept /* strengthened */` when _ctor-initializer_ is absent.